python/whylogs/api/logger/experimental/logger/actor/faster_fifo_queue_wrapper.py

```diff
@@ -0,0 +1,38 @@
+try:
+    from faster_fifo import Queue as FasterQueue  # type: ignore
+except ImportError:
+    raise ImportError("Install whylogs with extra [proc] for process based logging: pip install whylogs[proc]")
+from typing import Generic, List, Optional, TypeVar
+
+from whylogs.api.logger.experimental.logger.actor.actor import QueueWrapper
+
+_DEFAULT_QUEUE_SiZE = 1000 * 1000 * 1000
```

Comment:
It was pretty arbitrary. I have todos to make it configurable. I can easily add an env var if you think that isn't premature. I just picked a size that seemed reasonable in most cases (from my container testing). When the queue fills up then things start to back up. If the queue is full then the call to `log()` (and `send()` underneath the hood) will end up blocking for the queue to have enough room again.

The bigger size the more throughput is required to max it out. I went  bigger rather than small to minimize the main  process blocking.